### PR TITLE
HW2: Squashed and rebased commits

### PR DESCRIPTION
**Pull request by cgr2134**
Create nikefy.md

Create pandas.md

Update cgr2134.md

Update nikefy.md

### CODE REVIEW

File: students/cgr2134.md [hunk 1]
```diff
@@ -6,17 +6,11 @@ UNI: cgr2134
...
 Github: [link](https://github.com/cgr2134)
 
-## Project Proposal
+Github README: [link](https://github.com/cgr2134/cgr2134/blob/main/README.md)
 
-* Working on a python library that fetches data from Nike.com website, would be able to create metadata for a particular product and could be use to create large  
-* Simple game coded using Python
-* Some photography manipulation python library for photo files
+## Project Proposal
+* [nikefy](./projects/python/nikefy.md)
```
Comment (timkpaine) [inline students/cgr2134.md:12]: update path to `../` and re squash

File: students/cgr2134.md [hunk 2]
```diff
@@ -6,17 +6,11 @@ UNI: cgr2134
...
 
 ## Project Contributions
+* [pandas](./projects/python/pandas.md)
```
Comment (timkpaine) [inline students/cgr2134.md:15]: update path to `../` and re squash

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by cgr2134** [comment]
updated path to ../

File: students/cgr2134.md [hunk 3]
```diff
@@ -7,16 +7,8 @@ UNI: cgr2134
 Github: [link](https://github.com/cgr2134)
 
 ## Project Proposal
-
-* Working on a python library that fetches data from Nike.com website, would be able to create metadata for a particular product and could be use to create large  
-* Simple game coded using Python
-* Some photography manipulation python library for photo files
+* [nikefy](../projects/python/nikefy.md)
```
Comment (timkpaine) [inline students/cgr2134.md:10]: this file doesn't seem to exist

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/cgr2134.md (hunk 3 above)
Comment (cgr2134) [reply students/cgr2134.md:10]: created nikefy.md file should work now

**Pull request merged**